exec-service-wasmer/src/wasmer_executor.rs

```diff
@@ -29,20 +36,36 @@ impl WasmerExecutorData {
         self.opcode_cost.lock().unwrap().clone_from(opcode_cost);
         Ok(())
     }
+
+    pub(crate) fn get_vm_hooks_clone(&self) -> Rc<Box<dyn VMHooks>> {
```

Comment:
Is the `_clone()` suffix necessary? The method could be just `get_vm_hooks()` even if it makes a clone inside.

Reply:
agree, renamed